config/jobs/kubernetes/sig-cloud-provider/azure/release-1.23.yaml

```diff
@@ -389,7 +389,7 @@ periodics:
       - -c
       - >-
         kubectl apply -f templates/addons/azurefile-role.yaml &&
-        cd ${GOPATH}/src/sigs.k8s.io/azurefile-csi-driver && ./deploy/install-driver.sh master local,snapshot,enable-avset &&
+        cd ${GOPATH}/src/sigs.k8s.io/azurefile-csi-driver &&
```

Comment:
@andyzhangx is this change O.K. to the `capz-azure-file-vmss-1-23` job?

Reply:
why there is no `installCSIAzureFileDrivers` in this line?

Reply:
This is the generated spec for 1.23. Are we saying we need to change to this in generate.sh?:

```
if [[ "${release}" == "master" || "${release}" == "1.23" ]]; then
    installCSIAzureFileDrivers=" ./deploy/install-driver.sh master local &&"
fi
```

Reply:
if install azure file csi driver is performed, I think that's ok, thanks.